modules/client/examples/05-encoders-decoders/02-grant-with-condition.ts

```diff
@@ -15,36 +17,54 @@ import {
   GrantPermissionWithConditionParams,
   Permissions
 } from "@aragon/sdk-client";
-import { context } from '../index';
+import { context } from "../index";
 
-// Initializes a plugin context from the Aragon SDK context.
-const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
-// Initializes a general purpose client from the Aragon SDK context.
-const client: Client = new Client(contextPlugin);
+// Instantiates an Aragon OSx SDK client.
+const client: Client = new Client(context);
 
-const grantWithConditionParams: GrantPermissionWithConditionParams = {
+const params: GrantPermissionWithConditionParams = {
   who: "0x1234567890123456789012345678901234567890", // address to which the permission will be granted
   where: "0x2345678901234567890123456789012345678901", // where the permission is granted
   permission: Permissions.EXECUTE_PERMISSION, // the permission to grant. alternatively: UPGRADE_PERMISSION, SET_METADATA_PERMISSION, WITHDRAW_PERMISSION, SET_SIGNATURE_VALIDATOR, SET_TRUSTED_FORWARDER_PERMISSION, ROOT_PERMISSION, CREATE_VERSION_PERMISSION, REGISTER_DAO_PERMISSION, REGISTER_PERMISSION, REGISTER_ENS_SUBDOMAIN_PERMISSION, MINT_PERMISSION, MERKLE_MINT_PERMISSION, MODIFY_ALLOWLIST_PERMISSION, SET_CONFIGURATION_PERMISSION
```

Comment:
WITHDRAW permission, isn't it gone?

Reply:
Yes. Moreover, this is not the right place to provide and exhaustive list of all permissions but just an example. So I removed the list in 37d7db46960470bbb0946c48ba96865a86dabab1